src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -455,16 +464,39 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 		return errors.Wrapf(err, "Timeout while waiting for console to become available")
 	}
 
+	// Wait for OLM operators
+	if err = c.waitForOLMOperators(ctx); err != nil {
+		return errors.Wrapf(err, "Error while initializing OLM operators")
+	}
+
+	return nil
+}
+
+func (c controller) waitForOLMOperators(ctx context.Context) error {
+	// Get maximum wait timeout for OLM operators:
```

Comment:
We should have a check that will say of we have olm operators or not.
If we don't have olm operators we should exit immediately as this creates confusing behavior in assisted-service cause we are trying to do stuff that should not work in advance.

Reply:
Done.